src/Index.php

```diff
@@ -0,0 +1,288 @@
+        }
+
+        $options['httpParams'] = [
+            ':index' => $index
+        ];
+
+        $response = $this->kuzzle->query(
+            $this->kuzzle->buildQueryArgs('index', 'create'),
+            [
+                'index' => $index,
+                'body' => ['index' => $index]
+            ],
+            $options
+        );
+
+        return $response['result'];
+    }
+
+    /**
+     * Check if an index exist
+     *
+     * @param string $index
+     * @param array $options
+     *
+     * @return boolean
+     *
+     * @throws InvalidArgumentException
+     */
+    public function exists($index, array $options = [])
+    {
+        if (empty($index)) {
+            throw new InvalidArgumentException('Kuzzle\Index::exists: Unable to check if index exists: no index specified');
+        }
+
+        $options['httpParams'] = [
+            ':index' => $index
+        ];
+
+        $response = $this->kuzzle->query(
+            $this->kuzzle->buildQueryArgs('index', 'exists'),
+            [
+                'index' => $index
+            ],
+            $options
+        );
+
+        return $response['result'];
+    }
+
+    /**
+     * Delete an index
+     *
+     * @param string $index
+     * @param array $options
+     *
+     * @return mixed
+     *
+     * @throws InvalidArgumentException
+     */
+    public function deleteIndex($index, array $options = [])
+    {
+        if (empty($index)) {
+            throw new InvalidArgumentException('Kuzzle\Index::delete: Unable to delete index: no index specified');
+        }
+
+        $options['httpParams'] = [
+            ':index' => $index
+        ];
+
+        $response = $this->kuzzle->query(
+            $this->kuzzle->buildQueryArgs('index', 'delete'),
+            [
+                'index' => $index
+            ],
+            $options
+        );
+
+        return $response['result'];
+    }
+
+    /**
+     * Delete multiple index
+     *
+     * @param array $indexes
+     * @param array $options
+     *
+     * @return mixed
+     *
+     * @throws InvalidArgumentException
+     */
+    public function mDelete(array $indexes, array $options = [])
+    {
+        if (!is_array($indexes) || empty($indexes)) {
+            throw new InvalidArgumentException('Kuzzle\Index::mDelete: Unable to delete indexes: no indexes specified');
+        }
+
+        $response = $this->kuzzle->query(
+            $this->kuzzle->buildQueryArgs('index', 'mDelete'),
+            [
+              'body' => [
+                  'indexes' => json_encode($indexes)
+              ]
+            ],
+            $options
+        );
+
+        return $response['result'];
+    }
+
+
+    /**
+     * Retrieves the list of indexes stored in Kuzzle.
+     *
+     * @param array $options Optional parameters
+     * @return array of index names
+     */
+    public function listIndex(array $options = [])
```

Comment:
Why not ` list` like in the API and GO?
If for some reason we could not use `list`, I would rather go for `listIndexes` or `listIndices` (but I don't see why we could not use `list`)

Reply:
Because list is a reserved name in PHP 5.X so Travis Build will break.